src/Bridges/Constraint/bridges/square.jl

```diff
@@ -276,12 +276,37 @@ function MOI.get(::MOI.ModelLike, ::MOI.ConstraintSet, bridge::SquareBridge)
     return bridge.square_set
 end
 
+function MOI.supports(
+    model::MOI.ModelLike,
+    attr::Union{MOI.ConstraintPrimalStart,MOI.ConstraintDualStart},
+    ::Type{SquareBridge{T,F,G,TT,ST}},
+) where {T,F,G,TT,ST}
+    return MOI.supports(model, attr, MOI.ConstraintIndex{F,TT}) &&
```

Comment:
Just return `true`, `AbstractBridgeOptimizer` takes care of taking these into account

Reply:
Actually, I don't think this is true?